scriptExecutor.go

```diff
@@ -271,3 +274,36 @@ func buildDockerRunCommand(executionConf *executionConf) (string, error) {
 	return finalScript, nil
 
 }
+
+// Writes input vars to env file
+func writeToEnvFile(envMap map[string]string, filename string) error {
+	content, err := marshal(envMap)
+	if err != nil {
+		return err
+	}
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+	_, err = file.WriteString(content + "\n")
+	if err != nil {
+		return err
+	}
+	file.Sync()
+	return err
+}
+
+// Filters values of env variables on the basis of type and inserts to slice of strings
+func marshal(envMap map[string]string) (string, error) {
```

Comment:
when is this method going to return a non nil error?

Reply:
There was only one case where it could have been a non-nil err which was via 
`if d, err := strconv.Atoi(v); err == nil `
But as we're moving this to `isValidInteger` we no longer need to return any error